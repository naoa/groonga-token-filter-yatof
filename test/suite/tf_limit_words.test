register token_filters/yatof

table_create tf_limits TABLE_HASH_KEY ShortText
column_create tf_limits tf_limit COLUMN_SCALAR UInt32
load --table tf_limits
[
{"_key": "a", "tf_limit": 3}
]

tokenize TokenBigram "a a a a b b b b"--normalizer NormalizerAuto --token_filters TokenFilterTFLimit
