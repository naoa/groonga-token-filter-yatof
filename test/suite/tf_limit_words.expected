register token_filters/yatof
[[0,0.0,0.0],true]
table_create tf_limits TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
column_create tf_limits tf_limit COLUMN_SCALAR UInt32
[[0,0.0,0.0],true]
load --table tf_limits
[
{"_key": "a", "tf_limit": 3}
]
[[0,0.0,0.0],1]
tokenize TokenBigram "a a a a b b b b"--normalizer NormalizerAuto --token_filters TokenFilterTFLimit
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "a",
      "position": 0
    },
    {
      "value": "a",
      "position": 1
    },
    {
      "value": "a",
      "position": 2
    },
    {
      "value": "b",
      "position": 4
    },
    {
      "value": "b",
      "position": 5
    },
    {
      "value": "b",
      "position": 6
    },
    {
      "value": "b",
      "position": 7
    }
  ]
]
